src/js/game/LevelMVC/LevelView.js

```diff
@@ -1747,6 +1755,33 @@ module.exports = class LevelView {
       levelView.trees.push({ sprite: sprite, type: blockType, position: [x, y] });
     };
 
+    const buildDoor = (levelView, type) => {
+      atlas = this.blocks[blockType][0];
+      frame = this.blocks[blockType][1];
+      xOffset = this.blocks[blockType][2];
+      yOffset = this.blocks[blockType][3];
+      sprite = plane.create(xOffset + 40 * x, yOffset + plane.yOffset + 40 * y, atlas, frame);
+
+      frameList = [];
+      var animationFramesIron = Phaser.Animation.generateFrameNames(type, 0, 3, "", 1);
```

Comment:
nit: this (and all variables declared above) should be `consts`. I'd also slightly prefer to see the framelists kept separate for clarity as something like `openFrameList` and `closeFrameList` (and similarly for animationFramesIron), but that's probably not strictly necessary.

Reply:
oh, scratch the note about consts; I see now that these are declared above

Reply:
That code was simply copy pasted from how animations were handled in previous years. That can be refactored I suppose if we want if style or guidelines have changed? I might personally put that sort of refactoring in a future polish oriented PR? Let me know what your thoughts are on that.

As for all the variables being const (You're talking about the ones at the top of the function, like xOffset, atlas, and so on?) would that not cause problems with the fact that we're assigning values to them in several cases throughout the function? Is there some aspect of const variables in js that I'm unaware of? I assumed const would make them read-only

Reply:
Putting that in a polish-oriented PR is probably fine; like I said, it's a preference for the sake of future clarity (so that future copy-pasters get better code 😄) rather than a requirement.

And you're completely correct about the `consts`; I didn't realize I wasn't looking at the whole method. These are fine as-is